.env.example

```diff
@@ -11,6 +11,11 @@ FAKE_ADDRESS_SUGGESTIONS=true
 GOOGLE_RECAPTCHA_CONFIG='{"SITE_KEY":"6LcPRvQaAAAAAJRyxsFIB4rP5VH036pFOkNH8lgh", "SERVER_KEY":"6LcPRvQaAAAAADn_h1440Es7fXIGD0E4lpXR_FyF"}'
 HELP_REQUISITES='{ "support_email": "help@doma.ai", "support_email_mobile": "helpmobile@doma.ai", "bot_email": "service@doma.ai", "support_phone": "8 800 700-36-62" }'
 
+# Statsd metrics settings
```

Comment:
Its should not be required. May be you need to add it as commented example here.